doc/readme.html

```diff
@@ -288,6 +288,15 @@
 	  are usually found almost instantaneously, displaying the file
 	  name and line number of where the problem occurred.
 	</p>
+  <p>
+    By default <code>-Og</code> compiler flag is used. It promises most of the
+    debugging experience of <code>-O0</code> but at a moderate performance hit.
```

Comment:
But that's not right -- it promises *better* performance. Did you mean to reference `-O1` here?